prow/getting_started_deploy.md

```diff
@@ -57,21 +59,23 @@ See the [Next Steps](#next-steps) section after running this utility.
 
 If you do not want to use the `tackle` utility above, here are the manual set of commands tackle will run.
 
-Prow runs in a kubernetes cluster, so first figure out which cluster you want to deploy prow into. If you already have a cluster, skip to the next step.
-
-You can use the [GCP cloud console](https://console.cloud.google.com/) to set up a project and [create a new Kubernetes Engine cluster](https://console.cloud.google.com/kubernetes).
+Prow runs in a kubernetes cluster, so first figure out which cluster you want to deploy prow into.
+If you already have a cluster, skip to the **Create cluster role bindings** step.
```

Comment:
nit: Saying what to skip instead of what to skip to will be more robust to future changes to the document.